### ISSUE TITLE
Peer-review badge between reviews?

### ISSUE THREAD

**Issue by maelle** [ropensci/software-review-meta#100]
Via https://github.com/ropensci/comtradr/issues/45

A package will be extensively re-written and after that submitted again for peer-review.

In the meantime is it fine to "claim" the software-review badge? The badge links to the old thread that has a date but the badge might be misleading? Unless tempered with some "WIP" warning in the README?

**Comment by datapumpernickel** [comment]
Hi,

I think we are slowly approaching a moment, where we could possibly think about submitting to CRAN again. If there were another Peer-Review, which I would greatly appreciate, I think it would make sense to do it beforehand. 
Hence I wanted to ask, whether there is any news on this @maelle . :) 
Before starting the peer-review process, we will still have to clean up a little bit anyways. 

All the best!

**Comment by noamross** [comment]
We are happy to another review for the relaunch, @datapumpernickel!  This happens rarely but we have done re-reviews for major releases - such as for [*rebi*](https://github.com/ropensci/software-review/issues/29). 

I suggest that you re-submit as a regular submission, noting that this is for a re-release, and then update your badge to refer to the the new issue.

**Comment by datapumpernickel** [comment]
Thank you, that is great! We will then re-submit as regular submission as soon as we are ready!
